assets/templates/account/settings/settings_base.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'base.html' %}
+
```

Comment:
`settings` is double in the filename as the folder is already called `settings`.

So `account/settings/settings_base.html` should be `account/settings/base.html`, etcetera.

Reply:
what does a shorter name save? This way you can see a single html file and have some idea what it is intended for (with a file named base.html I assume it's the root of the project, not the root of a small part on a single template-chain). I also like template chains not call two different files with the same name if possible.

Reply:
It's clearer for reading, the path is also considered to be part of the file, you'll often see it how I described